homeassistant/components/binary_sensor/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 from homeassistant.helpers.entity_component import EntityComponent
 from homeassistant.helpers.entity import Entity
-from homeassistant.const import (STATE_ON, STATE_OFF)
+from homeassistant.const import (ATTR_SENSOR_CLASS, STATE_ON, STATE_OFF)
```

Comment:
I try to keep myself to these guidelines when it comes to constants: (I should write this down somewhere)
 - Only try to have constants in the const file that will be reused by multiple components.
 - It is totally fine for platforms to import constants from their parent component.
 - If an attribute is 'standardized' by a component (like sensor_class), add it as a property to the component base class that platforms can implement.